client/blocks/stats-navigation/intervals.js

```diff
@@ -43,17 +56,14 @@ const Intervals = ( props ) => {
 };
 
 Intervals.propTypes = {
-	className: PropTypes.string,
-	pathTemplate: PropTypes.string,
 	selected: PropTypes.string.isRequired,
+	pathTemplate: PropTypes.string,
+	className: PropTypes.string,
```

Comment:
Can `className` also be `undefined`? Seems like there's at least one such scenario:

https://github.com/Automattic/wp-calypso/blob/b359eb43a7aa2ed3d14381f683965e7f563eef05/apps/odyssey-stats/src/widget/mini-chart.tsx#L100

Reply:
Yes. I think prop types are optional by default.